freqtrade/templates/base_config.json.j2

```diff
@@ -13,6 +13,8 @@
     "fiat_display_currency": "{{ fiat_display_currency }}",{{ ('\n    "timeframe": "' + timeframe + '",') if timeframe else '' }}
     "dry_run": {{ dry_run | lower }},
     "cancel_open_orders_on_exit": false,
+    "trading_mode": "spot",
+    "margin_mode": null,
```

Comment:
I think this is pointless at the moment.

My expectation based on the "new config" title was that this would ask the user to select a trading mode.
As that's not the case, best remove these keys from the template for the moment. 

Even if they are in the template file, we cannot rely on the values to be there. 
If we need defaults - we have to set them in the schema in `constants.py`.